### ISSUE TITLE
Do we need/want a forum/mailing list ?

### ISSUE THREAD

**Issue by hreikin** [clearkimura/Customizer#49]
As suggested by @clearkimura in ISSUE https://github.com/clearkimura/Customizer/issues/43 i have set up a new issue so we can discuss if we need/want a new forum/mailing list

So an overview of what's been said : 

@mbnoimi suggested we need a forum or mailing list

@hreikin can set a forum up easily enough

@fluxer and @clearkimura stated they wouldn't have the time to answer each and every question which was raised in the forum/mailing list and asked who would be able to

I would be willing to answer any question i could on a daily basis but a forum is only as good as its community so i would probably need some help as i am no Linux master, just a googler :D

anybody else willing to comment/suggest anything further ?

**Comment by fluxer** [comment]
I can check the forums and help out only one day per week for now, my job does not allow me to spend more time.

**Comment by ghost** [comment]
I don't have reliable internet connection lately, so any online activities would be very limited by me.

And my current job takes much time that I can't get enough rest on weekends either.

I will continue to support as Customizer development team, but on irregular basis.

Say, if we really lack of volunteers to attend user submissions in forum.. I rather suggest to create a blog.

The blog shall allow public submission on Customizer how-to, test result and issues.
And blog needs fewer committees, while several of you be the editors to validate and attend the submitted post?

This way, anyone can do at least one-time commit and forget about attending each and every other user request. Because, the commit could cover the facing issue earlier.

In other words, provide known answers before questions were thrown. And let users browse and find their way.

Less commitment, but still does the job. If user needs interaction, turn on ''comment'' feature for particular post (don't turn on for any post by default to prevent spam).

Above all, just another suggestion. More suggestions are welcomed!

Sent using a Sony Ericsson mobile phone

**Comment by hreikin** [comment]
@clearkimura i did plan on copying over all issues to a section on the forum, as well as the user guide and any other info that is available on github, gets the same end result as your blog idea

i could check the forum daily and try to provide any answers/help, i am starting to get into a routine with my new job so i am finding my spare time again :D 

there are apps to view favourite forums available for smartphones which would make it easier for me (and possibly others) to keep track of the forum and provide any help required, or even just contact them till i am at a PC and can provide full help

**Comment by ghost** [comment]
@hreikin 
I see you are trying to collate isssues from user guide and GitHub, then put in one place--forum--which sounds good enough for troubleshooting purpose. 

Overall, the forum idea is okay. It may be just me, that this forum idea reminds me to Customizer old forum at Sourceforge.net, which had went silent by the time I first joined the Customizer Development team two years ago. I just hope the same thing won't be repeated.

> i did plan on copying over all issues to a section on the forum, as well as the user guide and any other info that is available on github, gets the same end result as your blog idea

In related matter, duplicate information shall _not_ be allowed.

As such, once you have moved the issues from user guide, I will remove these from user guide forever. This way, no confusion to users--always refer to one place.

**Comment by hreikin** [comment]
@clearkimura with the duplicate issues, i could lock that part of the forum so no user can post and just keep it updated with the issues from github/guide (once removed from guide it will be only github). i can place an announcement in that section with a link to the github issues page letting users know that is the place to open official issues

the forum could be for more general help to do with customising a distro such as the original issue in https://github.com/clearkimura/Customizer/issues/43

**Comment by ghost** [comment]
@hreikin 

>  i could lock that part of the forum so no user can post and just keep it updated with the issues from github/guide (once removed from guide it will be only github)

Then, I shall move "as it is" from user guide to github. After that, you can cross-link the post in forum with these to-be registered issues.

Regarding the issue submission standard, I have created a new issue for separate discussion. Refer to issue #50 for details.

> the forum could be for more general help to do with customising a distro..

Yes, that would be most welcomed by end users.

**Comment by hreikin** [comment]
@clearkimura let me know when this is done and I will start linking/transferring the info

I will start work on the forum again this weekend, any ideas where I can get a good quality logo for customizer for the forum, cant remember the exact size limit at the moment, I will edit the post later when at a pc and can check

**Comment by ghost** [comment]
@hreikin 

> let me know when this is done and I will start linking/transferring the info

Hopefully I can do by this weekend.

> any ideas where I can get a good quality logo for customizer for the forum

I asked the same question to @fluxer and unfortunately, the logo included with Customizer source code is the only logo available to this date.

Until last year, I managed to trace the source of tool image used in Customizer logo. It's found in one of common/public image domain site. And the quality was superb: SVG image format.

So left with the simple circle with black background and circular panned border can be easily created in any vector drawing application. And with little touching, it can be done (I think).

I was planning to reproduce the logo, but by the time I managed to trace the image source (can't recall how many times I refine my searching back then), I was too exhausted to proceed further. And since then, it was left undone.

Say, if you are familiar with basic vector drawing (Inkscape or LibreOffice Draw) and you can trace the said resource, you can proceed to recreate the similar logo from scratch.

Anyway, let me know, whether you want to recreate the logo or not.

**Comment by hreikin** [comment]
@clearkimura I finish work in 6 hours, i'll start finishing the forum later, I have used gimp before but not inkscape, I can always try with the logo and see if I can recreate it

**Comment by ghost** [comment]
@hreikin 

> let me know when this is done and I will start linking/transferring the info

I have finished moving official issues from user guide to GitHub. The official issues on GitHub has been updated with minor changes.

But, I haven't remove the issues from user guide yet. Will update the user guide later.

Anyway, you can start linking to these issues.

**Comment by hreikin** [comment]
@clearkimura just updated the forum layout will get on with linking/posting the issues to the forum later today, need some food and a little nap after work :D

**Comment by hreikin** [comment]
@clearkimura @fluxer i keep getting this error when trying to post : 

Forbidden

You don't have permission to access /posting.php on this server.

It happens a lot, if it continues and i cant find a workaround then i will switch to a different provider

**Comment by hreikin** [comment]
@clearkimura @fluxer i keep getting the same error and no luck finding a workaround :(

for now i have started making a new forum here : 

http://customizer.proboards.com/

please sign up again and i will make you mods, i will keep working on it to add more posts for an hour or 2 after work everyday depending how much time i get, hopefully it will be finished by the weekend

**Comment by ghost** [comment]
I found the posting.php error cause from their forum thread at http://support.freeforums.org/viewtopic.php?f=21&t=123179

Quoted as below:
it's the character '&' and some combinations, eg '<?', that cause the problem.

The freeforums works good, if we can avoid using special characters as name of the link.

Suggested workarounds:
1. Use name with alphanumeric characters only for hyperlink name i.e. ISSUE-2013-003.
2. Use online free shorten url generator with alphanumeric characters only
3. Don't include link on forum, the link to forum post is put under github comment instead

Actually, it is more difficult for me to access the proboards forum. It's more resource heavy and their mobile version is an Android/iOS apps (I don't own a smartphone).

If proboards forum is still prefered by you guys, I think I can't commit on forum except on weekends. That is, only when I go back to my hometown to access from my PC, which I can't tell when I could do so.

Sent using a Sony Ericsson mobile phone

**Comment by hreikin** [comment]
@clearkimura i have had problems when using only alphanumeric characters in the post title as well as having problems with the post content - eg when i copy and paste the issues. i changed the post title to "why wont it work" and it still wouldn't post it

rather than copying and pasting the issue i may have to give an overview and just include a link to the original github issue, also i may need to just name the issues by number in the post title to try avoid any problems

i start work soon so will have to look into it tomorrow morning when i finish

**Comment by hreikin** [comment]
@clearkimura @fluxer here is the forum : 

http://customizer.freeforums.org/index.php

all issues have been posted as well as the guide, if you can think of anything else that needs adding then let me know

i will get round to creating overviews of each issue in the individual posts it is just going to take a while :D

**Comment by ghost** [comment]
@hreikin 
Correct me if I am wrong.
The forum was created to allow discussions related to Customizer and remix building.

Somehow, I start to notice that we are duplicating the existing resources that GitHub has been hosting for all the while. Do we really need to do this?

**Comment by hreikin** [comment]
@clearkimura it is going to look like that until people use it, i can't populate it with threads from non existent users

if the forum is not needed or wanted just say so and it will be removed

i only created the issues section so it was there for people on the forum when they use it, you even agreed to it : 

"@hreikin

i could lock that part of the forum so no user can post and just keep it updated with the issues from github/guide (once removed from guide it will be only github)

Then, I shall move "as it is" from user guide to github. After that, you can cross-link the post in forum with these to-be registered issues."

**Comment by ghost** [comment]
@hreikin 

> if the forum is not needed or wanted just say so

I didn't mean that. If the community needs forum, we just proceed.

> i only created the issues section so it was there for people on the forum when they use it, you even agreed to it

True, I did agree and I have two reasons:
1. it would be easier for you to quote the link to GitHub issues, rather than quoting from the user guide itself (which may be subject to changes)
2. user guide should contain only static information; therefore moving to GitHub issues will make these issues easier to update without updating user guide again and again

Just that, we shouldn't duplicate things unnecessarily.

**Comment by hreikin** [comment]
@clearkimura as the forum stands now it is like your first point/reason, only links to github issue page

p.s. i don't know markdown :P

**Comment by ghost** [comment]
@hreikin 

> as the forum stands now it is like your first point/reason, only links to github issue page

Yes, I believe we have no problem with this. Only links.

> Just that, we shouldn't duplicate things unnecessarily.

What I mean by "duplicate" are these:
1. Announcements > General Information
   
   > The latest release is Customizer 3.2.3.
   > Customizer FAQ [..]
   > Customizer Official User Guide [..]
   > [..]
2. How to/User Guides > Official Customizer User Guide
   
   > The Official Customizer User Guide can be found here :
   > 
   > [https://docs.google.com/document/d/1Pfh ... qKTsA/edit](https://docs.google.com/document/d/1PfhHnSBjv-IDI7Yh5obhMGYCAV9Gw1NPEynU4GqKTsA/edit)
   > 
   > IT IS HIGHLY RECOMMENDED YOU READ THE USER GUIDE AND REFER TO IT BEFORE ASKING FOR HELP
3. Report a Bug/Issue > How to report a bug/issue
   
   > Below is the standard format for writing/resolving bugs/issues with Customizer, ALL bugs/issues should be reported on the official Github issue page for Customizer [..]
   > Quote:
   > To report issues, click New 
   >  [..]

These duplicated contents should be removed and summarized under "Announcement" category.
And it can be as simple as few lines:

> Dear users,
> 
> You can find general information and links to resources such as FAQ and user guide on [Customizer Wiki on GitHub](https://github.com/clearkimura/Customizer/wiki).
> 
> For bug and issue submission, you should refer to [this issue on GitHub](https://github.com/clearkimura/Customizer/issues/50).
> 
> For any forum related issues please contact micslinuxexp@gmail.com

This way, whenever GitHub Wiki content is updated you save a lot of hassles updating the thread on forum--because you already summarized in few words and include direct link to GitHub wiki (which already list few links readily).

Above all, it is important to remind ourselves that GitHub should always be central hub.
Extended discussions, tips for building a remix and such shall be on forum.
What do you think?

P.S.: You may highlight the Announcement thread description like, "You must read this thread first" to encourage users to always read the Announcement thread first.

**Comment by hreikin** [comment]
@clearkimura forum is set up as suggested however i have still linked to official user guide in how to section, i think it can only be a good thing for that to be there but if you don't then i will remove it

**Comment by ghost** [comment]
@hreikin 

> however i have still linked to official user guide in how to section, i think it can only be a good thing for that to be there but if you don't then i will remove it

How should I put this into words..
If you need to put the official user guide section, that is okay.
Direct link to the user guide file, isn't.
You are encouraged to link to Customizer Wiki on GitHub and explain, something like this:

> Dear users,
> 
> It is highly recommended you read the user guide and refer to it before asking for help.
> 
> The link to user guide can be found on [Customizer Wiki on GitHub](https://github.com/fluxer/Customizer/wiki).

Another thing is, please link to the Wiki using this URL:  https://github.com/fluxer/Customizer/wiki
This is not my repo originally.
The original URL will redirect to current ones.

**Comment by hreikin** [comment]
@clearkimura forum updated as suggested, anything else you would like to see ?

i can change the colour scheme if you would prefer a different look

**Comment by ghost** [comment]
@hreikin 
Many thanks for your update.

>  anything else you would like to see ?

No, I just wanted to point out about duplicates content (this is now resolved).
The rest is up to you.

**Comment by hreikin** [comment]
@clearkimura @fluxer i will make the last couple of posts tomorrow after work, after that it just needs linking to from github so people know it exists

if there are no users joining within the first month i may pull the forum down as the person who suggested the idea hasn't even commented on this issue so i doubt we will have many if any people posting at the forum :(

**Comment by ghost** [comment]
@hreikin @fluxer 
And GitHub Wiki has been updated with the link to forum.

**Comment by hreikin** [comment]
@clearkimura thanks

@mbnoimi the forum you requested has been set up and is ready

www.customizer.freeforums.org

please register

enjoy

**Comment by fluxer** [comment]
Since everything is up-and-running I guess you can close this issue.

**Comment by ghost** [comment]
@hreikin 
In case you didn't notice @fluxer comment several weeks ago..
I am closing this issue on your behalf.

@fluxer 
Closed as suggested.

**Comment by hreikin** [comment]
@clearkimura thankyou, i had seen the comment but didn't know how to close the issue and forgot to find out how to do it when i got back from work

**Comment by hreikin** [comment]
@clearkimura the forum has been removed due to inactivity, we still have 2 days before i cannot retrieve it, the question is do we want it ? do we need it ?

in my opinion, i don't think we want/need it as no-one has used it since it was set up, not even the person who suggested the forum in the first place :(

if you want the forum to stay then let me know and i will have to make sure something is posted regularly otherwise we will have this problem again

**Comment by ghost** [comment]
@mic
Hi, it has been a while. And apologies for delayed reply.

Hm.. I agree to your suggestion. We shall give up on forum. Anyway, we shall keep in touch sometimes.

I have been struggling between work and issues posted by several users on Github recently.

Customizer has many issues, due to lack of testing on newer release. Even after testing, hardly anyone volunteers to fix master and gambas3 branches (I haven't study nor do testing the devel branch either).

Based on recent issues' submissions, I suppose core features remain functional, though I haven't tested at all with 14.04 yet (I prefer LTS releases for my testing).

If you have experience to any of posted issues, you are welcomed to post comment or solution.

Once again, many thanks for organizing the forum until today.

Sent using a Sony Ericsson mobile phone

**Comment by hreikin** [comment]
@clearkimura 14.04 is an LTS release as far as i am aware. i will check through the issues later but can't promise i will be any help :( i only know what google tells me :P

**Comment by ghost** [comment]
@mic
14.04 is indeed LTS release. I mentioned my preference, while aware of this. Then again, I suppose that was an unintentional excuse. My bad.

I hardly use my own PC, even for testing mainly because of living and working far from home.

While my hands are itching to make progress on my remix based on 12.04 (the last release which works on non-PAE 32-bit CPU, except 64-bit has added support), I don't have much time for that either.

Back to your reply, it's okay if you don't know. I thought you have been working on remix using Customizer, which I presumed you may have experienced any similar issues.

I guess we are in the same boat.. Sigh.

Sent using a Sony Ericsson mobile phone

**Comment by hreikin** [comment]
Unfortunately yes we are, too much work not enough time to play

I'll try recreating a couple of the errors and try fixing them if i can
